source/includes/usage-examples/connect-code-examples.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+require 'vendor/autoload.php';
+
+use MongoDB\Client;
+
+// Connects to a local MongoDB deployment
+// start-local
+$uri = 'mongodb://localhost:27017/';
+$client = new Client($uri);
+// end-local
+
+// Connects to a MongoDB Atlas deployment
+// start-atlas
+$uri = '<Atlas connection string>';
+$client = new Client($uri);
+// end-atlas
+
+// Connects to a replica set
+// start-replica-set
+$uri = 'mongodb://<replica set member>:<port>/?replicaSet=<replica set name>';
+$client = new Client($uri);
+// end-replica-set
+
+// Connects to a MongoDB deployment and enables TLS using client
+// options
+// start-enable-tls-client
+$client = new Client(
+    'mongodb://<hostname>:<port>/',
+    ['tls' => true],
+);
+// end-enable-tls-client
+
+// Connects to a MongoDB deployment and enables TLS using connection URI
+// parameters
+// start-enable-tls-uri
+$uri = 'mongodb://<hostname>:<port>/?tls=true';
+$client = new Client($uri);
+// end-enable-tls-uri
+
+// Connects to a MongoDB deployment, enables TLS, and specifies the path to 
+// a CA file using client options
+// start-ca-file-client
+$client = new Client(
+    'mongodb://<hostname>:<port>/',
+    ['tls' => true, 'tlsCAFile' => '/path/to/ca.pem'],
+);
+// end-ca-file-client
+
+// Connects to a MongoDB deployment, enables TLS, and specifies the path to 
+// a CA file using connection URI parameters
+// start-ca-file-uri
+$uri = 'mongodb://<hostname>:<port>/?tls=true&tlsCAFile=/path/to/ca.pem';
+$client = new Client($uri);
+// end-ca-file-uri
+
+// Connects to a MongoDB deployment, enables TLS, and prevents OCSP endpoint checks
```

Comment:
It's not on the TLS guide so I don't have a section to link to, but I can create a ticket to add it to both the TLS page/landing page.

Reply:
That sounds reasonable. It probably warrants inclusion for any drivers that support it. Even without a dedicated example, it can be mentioned alongside `tlsDisableOCSPEndpointCheck`.